### ISSUE TITLE
Is the project dead?

### ISSUE THREAD

**Issue by webhdx** [cleverfox/stm32_followfocus#1]
Hello.
Are you still working on it? I was about to start hacking Zhiyun Motor as well and I was doing some research to not waste too much time on it. Did you reverse engineer the CMF-01 hardware? Do you have any block schematic how it is all connected internally?

**Comment by marklysze** [comment]
Checking in as well as the whether any progress has been made on this project or anyone has any details to communicate with the CMF-01 motor?

**Comment by cleverfox** [comment]
I spent about week trying to make motor spinning with good torque and precise angles, with no luck. I can't remember what was wrong. I have no crane gimbal, so I had no chance to see signals going to motor from original firmware. I had no problems with reversing CMF-01 hardware, PCB has only 2 layers and all tracks visible. I had schematic drawn by hand on sheet (except power nets), but this sheet was lost. If somebody has working CMF-01 with gimbal, oscilloscope and desire we can try to reanimate project

**Comment by marklysze** [comment]
Thanks for replying, I don't have the gimbal either, though may consider getting both if they become available for a reasonable cost. I have a logic analyser which may help. If I see any relevant info online I'll post it here

**Comment by cleverfox** [comment]
yeah, if you can get gimbal for few hours it can help. I live in small city, there is no photo/video rental services in nearby, and nobody of my friends own such gimbal

**Comment by Metacowboy** [comment]
A YT user got all the commands and build a android app to controll the focus motor via Bluetooth 
https://www.youtube.com/watch?v=plnVgDFxTQg
